questionnaire/questionnaire-service.js

```diff
@@ -350,7 +350,23 @@ function createQuestionnaireService({
         return progressEntryResource;
     }
 
+    function applicationSubmitted(submissionStatus) {
+        return submissionStatus !== 'NOT_STARTED';
```

Comment:
does this _categorically_ mean that it is submitted?

Reply:
As far as I am aware the application has been submitted in the sense that the Agree and submit" button has been clicked the application now no longer has a submission_status of 'NOT_STARTED' so submission HAS started the possible states post submission application could be in are [FAILED, IN_PROGRESS, COMPLETE].  
All of which represent post submission status therefore a confirmation page will be shown and the application session will be over.

Reply:
Made the change anyway, removed the function and added the conditional submissionStatus check within the code.